<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatchDetailsModal</class>
 <widget class="QDialog" name="MatchDetailsModal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="rulesList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>711</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>721</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string>The compatibility score is computed in three separate parts:
- computing the similarity between the client’s ideal animal (matching preferences) and the shelter animal.
- computing the similarity between the client’s personality and the animal’s personality
- and applying additional heuristics to consider the impact of client situation on compatibility.

The ideal animal similarity percentage is weighted to 40% of the total score, the personality similarity is weighted to 60% of the total score, and the heuristics add or minus 5% to the resulting score.

 The following shows the breakdown of computing the compatibility score. </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>731</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="animalName">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="clientName">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>COMPATIBILITY SCORE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="scoreLabel">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>711</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>This was the best match possible without creating another match whose animal or client would be better suited with this match's animal or client (Gale-Shapely). 
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
